sqlgen/mysql.go

```diff
@@ -80,6 +81,10 @@ func (q *SelectQuery) ToSQL() (string, []interface{}) {
 	buffer.WriteString(" FROM ")
 	buffer.WriteString(q.Table)
 
+	if len(q.Options.ForceIndex) > 0 {
```

Comment:
Have you considered also adding `USE INDEX()`? As this is an open-source library providing the option for both might be nice.

Reply:
OK, I just updated the PR to add `USE INDEX` as well 👍